src/components/MetadataPanel/style.css

```diff
@@ -0,0 +1,84 @@
+.container {
+    color: var(--dark-theme-text-color-bright);
+        padding: 16px 14px 16px 24px;
+}
+
+.collapsed {
+    padding: 14px 8px 8px 16px;
+}
+
+.agent-selected {
+    min-height: 265px;
+}
+
+.title {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 4px;
+}
+
+h3 {
```

Comment:
this is going to be applied to all h3 tags. this needs to be `.container h3`

Reply:
You're correct, thank you!